### PR TITLE
Bump Rails to 5.1.6

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by mmoll** [comment]
5.1.6 got released in the meanwhile :)

**Comment by ehelms** [comment]
Spec changes added for 5.1.6

File: vars/5.1.4-release.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
```
Comment (ekohl) [inline vars/5.1.4-release.yml]: Should the file be named ```release-5.1.4```?

**Comment by ekohl** [comment]
All individual packages are in, including https://github.com/theforeman/tfm-ror51-packaging/commit/23578656586ef41117e9e2db1a7641523a1d96d1 which makes me think this is now a redundant PR.

**Pull request closed without merging**